docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -288,8 +288,10 @@ appropriate flags on the Trac ticket based on the results of your review:
 "Patch needs improvement", "Needs documentation", and/or "Needs tests". As time
 and interest permits, mergers do final reviews of "Ready for checkin" tickets
 and will either commit the changes or bump it back to "Accepted" if further
-work needs to be done. If you're looking to become a merger, doing thorough
-reviews of contributions is a great way to earn trust.
+work needs to be done.
+
+If you're looking to become a member of the Review and Triage team, doing
```

Comment:
minor nit: seems to be "Triage and Review" everywhere else

Reply:
Thank you! Updated :star: